spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -64,14 +64,30 @@
       end
 
       context "with an email that already exists" do
-        let(:invited_user_email) { user.email }
+        context "and is confirmed" do
+          let(:invited_user_email) { user.email }
+
+          it "tells the user that the email has already been taken" do
+            expect {
+              click_on "Send invitation email"
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+          end
+        end
 
-        it "tells the user that the email has already been taken" do
-          expect {
-            click_on "Send invitation email"
-          }.to change { User.count }.by(0)
-          expect(InviteUseCaseSpy.invite_count).to eq(0)
-          expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+        context "and is not confirmed" do
+          let(:invited_user) { create(:user, :unconfirmed) }
```

Comment:
creating this user here creates it with an organisation. In the real life problem the user will not have an organisation so this test isn't quite right.

If the user already has an organisation, it means they have signed up or been invited so they should be skipped, land should probably get the message ""Email is already associated with an account. If you can't sign in, reset your password""